src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,229 @@
+{% extends "base/2019/base.html" %}
+
+{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web
+com os dados e as tendências do arquivo HTTP.{% endblock %}
+
+{% block twitter_image_alt %}{{ year }} Web Almanac{% endblock %}
+
+{% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
+{% block date_modified %}2020-07-27T00:00:00.000Z{% endblock %}
+
+{% block skip_navigation %}Pular navegação{% endblock %}
+
+{% block organization %}Web Almanac por HTTP Archive{% endblock %}
+{% block web_almanac_logo %}
+<span class="wa">Web Almanac</span>
+<span class="line-group">
+  <span class="pre">Por</span>
+  <span class="ha">HTTP Archive</span>
+</span>
+{% endblock %}
+
+{% block mission %}
+Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O
+Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web.
+É composto por 20 capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e
+distribuição.
```

Comment:
Combine these to keep line numbering same as English:

```suggestion
Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. É composto por 20 capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
```